src/driver/amdxdna/amdxdna_of_drv.c

```diff
@@ -75,6 +76,8 @@ static int amdxdna_of_probe(struct platform_device *pdev)
 		XDNA_WARN(xdna, "DMA configuration downgraded to 32bit Mask\n");
 	}
 
+	iommu_mode = AMDXDNA_IOMMU_NO_PASID;
+
```

Comment:
Why include amdxdna_devel.h and hardcoding iommu_mode to AMDXDNA_IOMMU_NO_PASID?